### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#469] [Esami & Esoneri M-Z]
### Traccia

<img width="689" alt="image" src="https://github.com/user-attachments/assets/858c0f02-8e00-417f-8bee-6fd6eaf64ec5">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/787f82db-0880-4cd5-bf98-134b1bf2f2f7)
